### PR TITLE
Upgrade typescript to 2.5.2 and support running tests on node 8

### PR DESCRIPTION
**Pull request by hadalin**
Closes https://github.com/genialis/resolwe-js/issues/39

### CODE REVIEW

**Comment by hadalin** [comment]
@lukaw3d Please test this locally.

File: package.json [hunk 1]
```diff
@@ -40,15 +41,13 @@
     "gulp-util": "^3.0.7",
     "is-json": "^2.0.1",
     "jasmine-core": "~2.4.1",
-    "karma": "^1.3.0",
-    "karma-coverage": "^1.0.0",
-    "karma-jasmine": "^1.0.2",
-    "karma-jspm": "^2.1.0",
-    "karma-junit-reporter": "^1.0.0",
-    "karma-phantomjs-launcher": "^1.0.0",
+    "karma": "^1.5.0",
```
Comment (lukaw3d) [inline package.json]: I had to explicitly install karma 1.5.0, because npm<5
Tested. Works.

Comment (hadalin) [reply package.json]: Changed to `"karma": "~1.5.0",`

**Comment by hadalin** [comment]
@kostko take note

**Pull request merged**